composer.json

```diff
@@ -145,6 +146,7 @@
 		],
 		"ci:php:cs-fixer": "php-cs-fixer fix --config .php-cs-fixer.php -v --dry-run --diff",
 		"ci:php:lint": "find .*.php *.php Classes Configuration Tests -name '*.php' -print0 | xargs -r -0 -n 1 -P 4 php -l",
+		"ci:php:mess": "phpmd Classes text Build/phpmd.xml",
```

Comment:
For consistency with the other configuration files (which have their own directory within `Build/`, please let's move the configuration file to `Build/phpmd/phpmd.xml`.